### DISCUSSION TITLE
Creating a LibRetro Frontend in Rust · RetroReversing

### DISCUSSION THREAD

**Discussion by giscus** [RetroReversing/retroReversing#121] [Post Comments]
# Creating a LibRetro Frontend in Rust · RetroReversing

Learn about RetroReversing, the home of retro game reverse engineering enthusiasts. Get resources, tutorials, and tools for analyzing vintage technology and join the community today!

https://www.retroreversing.com/CreateALibRetroFrontEndInRust

**Comment by rishooty** [comment]
Just letting you know this guide is fantastic! I've been using it to learn rust with the help of GPT to break the things down that i don't understand or fully understand in each step: https://github.com/rishooty/rustretro-tutorial

**Reply by RetroGameDeveloper** [reply]
So happy to hear this! This was written as notes as I went along so there are a bunch of things that need to be fixed/rewritten. Any help on fixing the bugs in this article are much appreciated!
